draft-polli-ratelimit-headers.md

```diff
@@ -766,21 +766,6 @@ and the `RateLimit-Reset` value should not be considered fixed nor constant.
 Subsequent requests may return an higher `RateLimit-Reset` value to limit
 concurrency or implement dynamic or adaptive throttling policies.
 
-## Resource exhaustion and clock skew
-
-Implementers returning `RateLimit-Reset` must be aware that many throttled
-clients may come back at the very moment specified.
-For example, when returning
-
-~~~
-RateLimit-Reset: Tue, 15 Nov 1994 08:00:00 GMT
-~~~
-
-there's a high probability that all clients will show up at `08:00:00`.
-
-This could be mitigated adding some jitter to the header value.
```

Comment:
I think this advice still holds even if using delta seconds. Worth keeping?

Reply:
Thought about that and yes. But rephrasing a bit

Reply:
is that ok?

Reply:
Yes, looks good!